Hymns/Assets.xcassets/empty favorites illustration.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "favorites illustration.png",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
```

Comment:
You only provided a file for the 1x version. How does the 2x and 3x work? You don't need to provide images for those also?

Reply:
It's wierd. I have tried creating 2x and 3x versions for the 1x and it always looks pixelated and wrong. So then I tried making the initial image the 3x and downgrading by putting in 1x and 2x but it also looks pixelated. It seems to me just by leaving the highest res image we are getting the best result on all devices. I have tried different ways and websites to scale out 2x and 3x but with no luck. To me what we have is looking good on every test emulator so idk...

Reply:
ok, I guess we can try to fix it later if we need to